dev-python/confuse/confuse-1.4.0.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} pypy3 )
+inherit distutils-r1
+
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
+DESCRIPTION="Confuse is a configuration library for Python that uses YAML"
+HOMEPAGE="https://github.com/beetbox/confuse"
+SRC_URI="https://github.com/beetbox/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	dev-python/pyproject2setuppy[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? ( dev-python/sphinx )
+	test? ( dev-python/nose[${PYTHON_USEDEP}] )
```

Comment:
This isn't wrong, but you can also use `distutils_enable_tests` which adds many other things you've already specified in this ebuild.
https://dev.gentoo.org/~mgorny/python-guide/distutils.html#enabling-tests

Reply:
Using distutils the result is much more clean :)